plugins/field-angle/src/field_angle.ts

```diff
@@ -658,12 +658,17 @@ export class FieldAngle extends Blockly.FieldNumber {
   }
 }
 
-// Unregister legacy field_angle that was in core.  Delete this once
-// core Blockly no longer defines field_angle.
-// If field_angle is not defined in core, this generates a console warning.
-Blockly.fieldRegistry.unregister('field_angle');
+/**
+ * Register the field and any dependencies.
+ */
```

Comment:
If formatter + linter allow, consider:

```suggestion
/** Register the field and any dependencies. */
```

Reply:
Done.